emscripten.py

```diff
@@ -112,11 +114,11 @@ def update_settings_glue(metadata, DEBUG):
     shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE = []
 
   all_funcs = shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE + [shared.JS.to_nice_ident(d) for d in metadata['declares']]
-  implemented_funcs = [x[1:] for x in metadata['implementedFunctions']]
-  shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE = sorted(set(all_funcs).difference(implemented_funcs))
+  shared.Settings.DEFAULT_LIBRARY_FUNCS_TO_INCLUDE = sorted(set(all_funcs).difference(metadata['exports']))
```

Comment:
this part of the change is not 100% obvious to as correct, but it tests pass it is all good.

Reply:
The `implemented_funcs` local variable here was effectively the unmagled versions `metadata['implementedFunctions']` but `metadata['implementedFunctions']` itself was that mangled version of `metadata['exports']` so I think it should be equivalent, no?  This allows be to remove the fake `metadata['implementedFunctions']` setting.